__tests__/suites/card-button-text.test.ts

```diff
@@ -4,18 +4,19 @@ import { FUNDING } from "@paypal/sdk-constants";
 import { ButtonsComponent } from "../components/buttons-component";
 
 const buttonTextByLanguage: Record<string, string> = {
-    "en-US": "Debit or Credit Card",
-    "de-DE": "Debit- oder Kreditkarte",
+    en: "Debit or Credit Card",
+    de: "Debit- oder Kreditkarte",
 };
 
+/* Returns a 2 char string representing the language, i.e. "en", "de" */
```

Comment:
Please use single line comment for this.
```suggestion
// Returns a 2 char string representing the language, i.e. "en", "de"
```